lib/Sitemap/SitemapPageRepository.php

```diff
@@ -50,13 +56,29 @@ private function getUrlsFromFiles(string $path, string $extension = 'html') : ar
             RecursiveDirectoryIterator::SKIP_DOTS | RecursiveDirectoryIterator::CURRENT_AS_PATHNAME
         );
 
+        $it = new RecursiveIteratorIterator($it);
+
+        $sortedIterator = new class($it) extends SplHeap {
```

Comment:
What's this contraption for? O_o

Reply:
Do we need a sorted result?

Reply:
Lolll I didn't want it but I added it to get a consistent order for the test. I'm sure there is a better way but this is where I landed last night.

Reply:
I'd be OK with just a comment explaining this :+1:

Reply:
Thinking more about it this morning, I could move the sort to the test before asserting. I was getting a different order locally compared to travis. We could still get a different order of the URLs in the sitemap depending on where the site is built but we only build and push to github on the build server. What do you think?

Reply:
If you use custom files for your test cases, you can be (pretty) sure about the order in your tests by the file names you choose. If we don't want empty test files in our repository, we can introduce mikey179/vfsStream to replace the filesystem in the tests.

If you want to keep this logic, try `SplMaxHeap` or `SplMinHeap` for the sorting instead of your custom heap. Maybe one of those do the trick for you too.